plugins/versionpress/tests/Automation/WpAutomation.php

```diff
@@ -557,12 +557,17 @@ private function prepareFiles()
     private function ensureCleanInstallationIsAvailable()
     {
 
-        if (!is_dir($this->getCleanInstallationPath())) {
-            $downloadPath = $this->getCleanInstallationPath();
-            FileSystem::mkdir($downloadPath);
+        $cleanInstallationPath = $this->getCleanInstallationPath();
+
+        if (!is_dir($cleanInstallationPath) || !is_file($cleanInstallationPath . '/wp-settings.php')) {
+            if (!is_dir($cleanInstallationPath)) {
```

Comment:
These conditions seem a bit ugly to me, if you have a better idea of how to implement them feel free to, @JanVoracek.